Script/AtomicEditor/hostExtensions/HostExtensionServices.ts

```diff
@@ -0,0 +1,171 @@
+//
+// Copyright (c) 2014-2015, THUNDERBEAST GAMES LLC All rights reserved
+// LICENSE: Atomic Game Engine Editor and Tools EULA
+// Please see LICENSE_ATOMIC_EDITOR_AND_TOOLS.md in repository root for
+// license information: https://github.com/AtomicGameEngine/AtomicGameEngine
+//
```

Comment:
Oops, I missed the licensing block in the original PR.  Would you mind switching these to the MIT block in a followup?

For example: https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Script/AtomicEditor/main.ts#L1